enterprise/api/v1/views.py

```diff
@@ -343,12 +345,9 @@ def _revoke_enrollment(self, enrollment_api_client, enterprise_enrollment, cours
             course_overview (CourseOverview): The course overview object associated with the enrollment. Used
                 to check for course completion.
         """
-        if self._has_user_completed_course_run(enterprise_enrollment, course_overview):
-            return
-
         course_run_id = course_overview.get('id')
         enterprise_customer_user = enterprise_enrollment.enterprise_customer_user
-        audit_mode = CourseModes.AUDIT
+        audit_mode = CourseMode.AUDIT
```

Comment:
What if `CourseMode` is `None` due to the try/except with the import?

Reply:
This is only called from the `license_revoke` action method in this viewset.  The first thing it does is make sure that `get_course_overviews, get_certificate_for_user, CourseMode` are all truthy, and errors out otherwise.